pkgs/servers/sql/postgresql/ext/postgis.nix

```diff
@@ -43,14 +45,13 @@ stdenv.mkDerivation rec {
     sed -i "s|\$(DESTDIR)\$(PGSQL_BINDIR)|$prefix/bin|g
             " \
         "raster/scripts/python/Makefile";
-  '';
-
-  preInstall = ''
     mkdir -p $out/bin
+    ln -s ${postgresql}/bin/postgres $out/bin/postgres
   '';
 
   # create aliases for all commands adding version information
   postInstall = ''
+    rm $out/bin/postgres
```

Comment:
What's the reason for linking Postgres in postConfigure only to remove it in postInstall?

Reply:
Never mind, I just saw the reason in the PR description :man_facepalming:. Perhaps a comment in the code to highlight the issue?